Spectrum_Peak_Visualization.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import peakutils
+import SPEFile
+
+"""
+plot_spectrum plots the spectra only.
+"""
+def plot_spectrum(spectrum, title,energy_range='None'):
+    zero_offset = spectrum.energy_cal[0]
+    energy_per_channel = spectrum.energy_cal[1]
+    energy_axis = zero_offset + energy_per_channel*spectrum.channel
+    counts = spectrum.data
+    
+    #Graphs the spectrum first without any ROIs.
+    plt.semilogy(energy_axis,counts,drawstyle='steps-mid')
+    plt.xlabel('Energy (keV)')
+    plt.ylabel('Counts')
+    plt.title(title)
+    
+    """
+    If energy_range is set to a tuple of two values, then below code will graph
+    the spectrum within the energy_range values for the energies (x-values).
+    """    
+    if energy_range != 'None':
+        plt.xlim([energy_range[0], energy_range[1]])
+
+
+"""
+plot_peaks plots spectra with peaks highlighted.
+"""
+def plot_peaks(spectrum,title,energy_range='None'):
+    zero_offset = spectrum.energy_cal[0]
+    energy_per_channel = spectrum.energy_cal[1]
+    energy_axis = zero_offset + energy_per_channel*spectrum.channel
+    counts = spectrum.data
+    
+    #Graphs the spectrum first without any ROIs.
+    plt.plot(energy_axis,counts,drawstyle='steps-mid')
+    plt.xlabel('Energy (keV)')
+    plt.ylabel('Counts')
+    plt.title(title)
+    
+    """
+    Need to call Jackie's peak finder function to get indices of the peaks and
+    place those indices in a list for use in highlighting ROIs below. 
+    Currently, Chris's code, below, is being used for testing purposes for 
+    highlighting peaks.
+    """
```

Comment:
Find a way to take the ROIs as inputs, rather than finding them dynamically with a peak finder.
(Besides, code shouldn't be copied, see below.)